lib/account_deleter.rb

```diff
@@ -69,10 +68,6 @@ def delete_standard_person_associations
     end
   end
 
-  def disconnect_contacts
```

Comment:
Ah, I see, I missed that before, but I made a little change in `DataGenerator` for #6750 which actually created a double-sided contact: https://github.com/diaspora/diaspora/pull/6750/files#diff-4af80ea1ce63ed0a143b1b026bdb7eec

When I made methods for `DataGenerator` that are used in the account deletion spec I followed the spec that was there before. This spec didn't create any double-sided contacts. There are methods in `DataGenerator` which can make a double-sided contact but I didn't call them for the account deletion spec, only for exporter. But with the change in #6750 it is now different.